.cbmc-batch/source/proof_allocators.c

```diff
@@ -16,14 +16,16 @@
 #include "proof_allocators.h"
 #include <stdlib.h>
 
-//Use the same functions as the standard default allocator, but nondeterministically have malloc
-//return null.  This is needed because the CBMC model of malloc cannot fail, so we cannot
-//test the null case otherwise.
+// Use the same functions as the standard default allocator, but nondeterministically have malloc
+// return null.  This is needed because the CBMC model of malloc cannot fail, so we cannot
+// test the null case otherwise.
 static void *can_fail_malloc(struct aws_allocator *allocator, size_t size) {
```

Comment:
What is the point of the aws_allocator?  Is it never used in these proofs?

Reply:
This file was from a previous PR.